### PR TITLE
Enable the ability to use the PHP development server.

### PR DESCRIPTION
**Pull request by beryllium**
This detects `cli-server` mode and passes through calls to physical files, so that CSS/JS/Img assets can be served.

`php -S0.0.0.0:8000 -t public public/index.php`

Some links go wonky and you have to make sure `/etc/hosts` is configured properly to match your `APP_DOMAIN`, but it works. Oh, and SSL is not supported. `¯\_(ツ)_/¯`

### CODE REVIEW

File: public/index.php [hunk 1]
```diff
@@ -1,5 +1,10 @@
 <?php
 
+$filename = __DIR__.preg_replace('#(\?.*)$#', '', isset($_SERVER['REQUEST_URI']) ? $_SERVER['REQUEST_URI'] : '');
```
Comment (snipe) [inline public/index.php:3]: I don't really like adding code that's used solely for development purposes, especially when things like Valet and Homestead exist. Did you try with Valet/Homestead and were having trouble?

Comment (beryllium) [reply public/index.php:3]: I am unfamiliar with Laravel and prefer to do local development versus Homestead/similar solutions.

**Comment by snipe** [comment]
Homestead is local, it's just a local vagrant instance with a little extra laravel-goodness.

https://laravel.com/docs/5.4/valet
https://laravel.com/docs/5.4/homestead

**Comment by beryllium** [comment]
Oh! I must've been thinking of Forge/Heroku.

**Comment by beryllium** [comment]
Turns out the `php artisan serve` command handles this. Never mind! :)

**Pull request closed without merging**